src/sentry/api/serializers/models/alert_rule_trigger_action.py

```diff
@@ -23,6 +27,12 @@ def human_desc(self, action):
         elif action.type == action.Type.OPSGENIE.value:
             return "Send an Opsgenie notification to " + action.target_display
         elif action.type == action.Type.DISCORD.value:
+            if not action.target_display:
+                logger.info(
+                    "discord.action.description.no.channel",
+                    extra={"target_identifier": action.target_identifier},
+                )
+                action.target_display = ""
```

Comment:
@17hogeju do we have to mutate the action this way?

Reply:
I thought about just having f"Send a Discord notification to {action.target_display or ''}" since that would handle the null. I added in the if statement solely for logging and threw in the null string in the same step just to keep the line below the same. I could just use the f string and remove the mutation.

Reply:
@17hogeju better to not mutate if we can